deploy/dev/5_load_environment.sh

```diff
@@ -8,9 +8,20 @@ function main() {
 
   set_namespace "$APP_NAMESPACE_NAME"
 
-  configure_secret
+  if [ "${DEV}" = "false" ]; then
+    configure_secret
 
-  deploy_env
+    deploy_env
+  else
+    setup_helm_environment
+
+    create_k8s_secret
+
+    deploy_chart
+
+    deploy_helm_app
+
+  fi
 }
 
 function configure_secret() {
```

Comment:
you can remove the `function` keyword throughout the repo

Reply:
the name is quote misleading as we're creating the secret and not configuring it. so `create_k8s_secret` seems more fitting. but then, we already have this function. this leads to the real question - what's the difference between the two? can we give them proper names so that it is clear?

Reply:
I commented on the difference above. I can rename the helm one to `create_k8s_secret_for_helm_deployment`